WordPress/WordPressData/Swift/AbstractPost.swift

```diff
@@ -10,7 +11,8 @@ extension AbstractPost {
     /// Returns `true` if the post was never uploaded to the remote and has
     /// not revisions that were marked for syncing.
     var isNewDraft: Bool {
-        wpAssert(isOriginal(), "Must be called on the original")
+        // FIXME: Delaying wpAssert migration because of underlying dependencies
+        // wpAssert(isOriginal(), "Must be called on the original")
```

Comment:
This and all the others are addressed by https://github.com/wordpress-mobile/WordPress-iOS/pull/24346